tests/end-to-end/regression/5592/Issue5592Test.php

```diff
@@ -14,19 +14,25 @@
 use function set_error_handler;
 use function set_exception_handler;
 use PHPUnit\Framework\TestCase;
+use PHPUnit\Runner\ErrorHandler;
+use Throwable;
 
 class Issue5592Test extends TestCase
 {
     public function testAddedAndRemovedErrorHandler(): void
     {
-        set_error_handler(static fn () => false);
+        $previous = set_error_handler([$this, 'addedAndRemovedErrorHandler']);
```

Comment:
using a named error handler, so its easier to point out which error handler violates assertions.

before this PR we only got `Closure` as a error handler, which is hard to use for debugging, as we have multiple sites registering a `Closure`.